### ISSUE TITLE
Consolidate application registration example

### ISSUE THREAD

**Issue by rpwagner** [globusonline/globus-integration-examples#5]
Extract the "Register Your Application with Globus Auth" portion of the WordPress doc and move it to its own page, or maybe to the front page. Later this move this into the main docs site.

Add walk-thru of each relevant app type, each on its own page:

- Confidential App
- Native App

**Comment by lliming** [comment]
I've submitted a PR to the Auth developer's guide (PR#216 in globusonline/docs.globus.org) that replaces section 1.1 "Registering an Application" with more complete content. 

The content was drafted in a Google Doc at https://docs.google.com/document/d/1vyjnzSr9W6rmQWv68pmUm4TGGvMHykF82-jZkf8rMpE/edit?usp=sharing . 

I forked the docs repo, made my change, and submitted the PR with instructions to notify the Auth development team when it's available in staging so they can conduct their own review, per Rachana's instructions. 

We're now waiting on the docs repo maintainers to merge the PR and the Auth development team to review the content. When they push the changes to production, we can reference the Auth developer's guide instead of writing our own instructions.
